tests/stub/routing.py

```diff
@@ -404,7 +499,7 @@ def read_script_with_bookmarks(self):
         !: AUTO GOODBYE
         !: AUTO RESET
 
-        C: RUN "RETURN 1 as n" {} {"mode": "r", "db": "adb", "bookmarks": ["sys:1234", "foo:5678"]}
+        C: RUN "RETURN 1 as n" {} {"mode": "r", "db": "adb", "bookmarks{}": ["sys:1234", "foo:5678"]}
```

Comment:
is it intentional ?

Reply:
Yes. It allows the bookmarks to be sent in arbitrary order.